.storybook/stories/tabs/tabs-collapsed.stories.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2016-2023 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { ClrTabs, ClrTabsModule } from '@clr/angular';
+import { Parameters } from '@storybook/addons';
+import { Story } from '@storybook/angular';
+
+import { TabsLayout } from '../../../projects/angular/src/layout/tabs/enums/tabs-layout.enum';
+import { setupStorybook } from '../../helpers/setup-storybook.helpers';
+
+const collapsedStory: Story = args => ({
+  template: `
+    <clr-tabs [clrLayout]="clrLayout">
+      <clr-tab *ngFor="let _ of createArray(tabCount); let i = index">
+        <button clrTabLink>{{title}} {{i + 1}}</button>
+        <clr-tab-content *clrIfActive="activeTab === (i + 1)">
+          <p>
+            {{content}} {{i + 1}}
+          </p>
+        </clr-tab-content>
+      </clr-tab>
+    </clr-tabs>
+  `,
+  props: { ...args },
+});
+
+const defaultParameters: Parameters = {
+  title: 'Tabs/Tabs - Reactive',
+  component: ClrTabs,
+  argTypes: {
+    // inputs
+    clrLayout: { defaultValue: TabsLayout.HORIZONTAL, control: { type: 'inline-radio', options: TabsLayout } },
+    // methods
+    closeOnEscapeKey: { control: { disable: true }, table: { disable: true } },
+    closeOnFocusOut: { control: { disable: true }, table: { disable: true } },
+    closeOnOutsideClick: { control: { disable: true }, table: { disable: true } },
+    openOverflowOnFocus: { control: { disable: true }, table: { disable: true } },
+    resetKeyFocusCurrentToActive: { control: { disable: true }, table: { disable: true } },
+    toggleOverflowOnClick: { control: { disable: true }, table: { disable: true } },
+    toggleOverflowOnPosition: { control: { disable: true }, table: { disable: true } },
+    // story helpers
+    createArray: { control: { disable: true }, table: { disable: true } },
+    tabCount: { control: { type: 'number', min: 1, max: 100 } },
+    activeTab: { control: { type: 'number', min: 1, max: 100 } },
+  },
+  args: {
+    // story helpers
+    createArray: n => new Array(n),
+    tabCount: 4,
+    activeTab: 1,
+    title: 'Tab',
+    content: 'Tab Content',
+  },
+  viewport: {
+    defaultViewport: 'large',
+  },
+};
+
```

Comment:
Instead of creating a new story is it possible to make modifications on original one to meet viewport changes?

Reply:
I have create a ticket for that as discussed